report_thesis/src/sections/related_work.tex

```diff
@@ -49,9 +49,17 @@ \subsection{Preprocessing and Feature Engineering}
 \subsection{Dimensionality Reduction Techniques}
 Dimensionality reduction techniques such as \gls{pca} play a crucial role in managing the high-dimensional nature of \gls{libs} data.
 
-\citet{rezaei_dimensionality_reduction} explored various machine learning techniques combined with \gls{pca} for dimensionality reduction. Their results demonstrated that \gls{svr} with \gls{pca} performed the best for different elements, highlighting the effectiveness of dimensionality reduction techniques in improving model performance by managing the complexities of high-dimensional data.
+\cite{pca_review_paper} conducted a comprehensive review of \gls{pca} applied in a \gls{libs} context. 
+Their review highlighted several studies that succesfully deployed \gls{pca}.
+For example, \cite{moncayo_pca} used \gls{pca} to analyze megapixel elemental maps composed of more than 1 million LIBS spectra. 
+\gls{pca} successfully separated the contributions of various minerals, including those present in low amounts, highlighting the power of \gls{pca} in handling highly diverse samples.
+Another example is \citet{porizka_pca} used \gls{pca} to filter outliers and classify samples according to their matrix. 
+This was followed by a univariate calibration of copper (\ce{Cu}) in soil samples, which resulted in a reduced bias. 
+Rather than mitigating the matrix effect, the use of PCA avoided it altogether.
```

Comment:
They claim that it avoided...?

Reply:
I guess I could make this a stronger point. They basically said that they completely mitigated the matrix effects. I will rewrite.